test/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required(VERSION 3.9)
 set(CMAKE_CXX_STANDARD 11)
 
+if(WIN32)
+  set(CMAKE_CXX_FLAGS "/W4 /WX /EHsc")
```

Comment:
This works fine too, but if you want to do it the "correct" CMake way, you would either use `target_compile_options` on the executable targets below, or set `add_compile_options` for global modifications.

Reply:
@jviotti done, PTAL